File: app/src/main/java/com/hanix/randomgame/view/tab/object/ObjectActivity.java
package com.hanix.randomgame.view.tab.object;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.hanix.randomgame.R;
import com.hanix.randomgame.view.event.OnSingleClickListener;

public class ObjectActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_object);

        init();
    }

    /**
     * findViewById 및 OnClickListener
     */
    private void init() {

    }

    /**
     * onClickListener -> 중복 클릭 방지를 위해 OnSingleClickListener 사용
     */
    OnSingleClickListener objectClick = new OnSingleClickListener() {
        @Override
        public void onSingleClick(View v) {

        }
    };
}

File: app/src/main/java/com/hanix/randomgame/view/tab/object/ObjectActivity.kt
package com.hanix.randomgame.view.tab.`object`

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.hanix.randomgame.R
import com.hanix.randomgame.view.event.OnSingleClickListener

class ObjectActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_object)
        init()
    }

    /**
     * findViewById 및 OnClickListener
     */
    private fun init() {}

    /**
     * onClickListener -> 중복 클릭 방지를 위해 OnSingleClickListener 사용
     */
    var objectClick: OnSingleClickListener = object : OnSingleClickListener() {
        override fun onSingleClick(v: View?) {}
    }
}